docs/FAQ.md

```diff
@@ -0,0 +1,10 @@
+<!--
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+-->
+
+## FAQ
+
+#### Which browsers is Accessibility Insights for Web available for?
+
+Accessibility Insights for Web is curently available on Google Chrome only.
```

Comment:
Ah, sorry I missed this yesterday: there's a typo - curently should be currently.
```suggestion
Accessibility Insights for Web is currently available on Google Chrome only.
```